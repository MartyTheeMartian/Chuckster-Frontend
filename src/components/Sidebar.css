.sidebar-ctn {
  width: 280px;
  height: 100vh;
}

.title-ctn {
  font-size: 18px;
  font-weight: 700;
}
