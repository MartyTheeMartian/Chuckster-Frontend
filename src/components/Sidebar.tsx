import 'Sidebar.css';

export default function SideBar(): JSX.Element {
  return (
    <nav className='sidebar-ctn'>
      <div className='title-ctn'>
        Chuckster &#128513;
      </div>
    </nav>
  );
};
