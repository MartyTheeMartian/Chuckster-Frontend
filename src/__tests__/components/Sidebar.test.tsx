import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Sidebar from 'components/Sidebar';

it('renders App component', () => {
  const result = shallow(<Sidebar />);
  expect(toJson(result)).toMatchSnapshot();
});
