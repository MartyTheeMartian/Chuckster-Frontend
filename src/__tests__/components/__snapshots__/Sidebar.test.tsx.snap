// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<nav
  className="sidebar-ctn"
>
  <div
    className="title-ctn"
  >
    Chuckster 😁
  </div>
</nav>
`;
